types/pica/index.d.ts

```diff
@@ -0,0 +1,95 @@
+// Type definitions for pica 5.1
+// Project: https://github.com/nodeca/pica
+// Definitions by: Hamit YILMAZ <https://github.com/hmtylmz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class Pica {
```

Comment:
Shouldn't this be exported? I get `File '/node_modules/@types/pica/index.d.ts' is not a module.  TS2306` if nothing is exported from this file.

Reply:
Pica is a UMD module (meaning it works as a global, an AMD module, or a CommonJS module). So while it’s not exactly _wrong_ to simply declare it as a global, it’s incomplete. There should be an `export as namespace Pica` to indicate it’s available as UMD.